### ISSUE TITLE
FEEDBACK: GeoSearch across multiple locators

### ISSUE THREAD

**Issue by tedrick** [tedrick/Survey123-Testing#156]
The map geosearch functionality will now simultaneously search across all of the geocodes configured in the organization settings.  Please attempt to do place searches and provide feedback on the experience.

**Comment by tedrick** [comment]
Sarah Bubka

**Comment by sbubka94** [comment]
When switching from "Search everywhere" to "Search within the visible map extent" the current search does not honor these settings. For example...

I chose "search everywhere" and my "San Diego" locator. I searched the Convention Center and got some suggestions. While the suggestions are still open, I switch from "search everywhere" to "Search within the visible extent" and the addresses for Convention Center do not clear. I even tried deleting the entire phrase, and re-typing Convention Center but it still suggested the same addresses, despite my map extent not being SD.

http://www.screencast.com/t/gTtSmW8fJmd

I think I would expect that the suggestions for Convention Center would disappear as I specified the visible map extent. Even if you
